### PR TITLE
Relational database

### PR DESCRIPTION
**Pull request by sgupta1409**
Successfully Created Relational Database

### CODE REVIEW

File: connection.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
-import { Sequelize } from 'sequelize';
+import Sequelize from 'sequelize';
 
-// Option 3: Passing parameters separately (other dialects)
-const sequelize = new Sequelize('postgres://postgres:qod@123@localhost:5432/user'); // Example for postgres
+const sequelize = new Sequelize('UserInformation', 'user_1', 'test123', {
+  host: 'localhost',
```
Comment (DjangoUnchainedd) [inline connection.js:4]: Use env file for config, don't hardcode it.

File: test/test.js [hunk 2]
```diff
@@ -16,36 +19,79 @@ describe('GET /user', () => {
...
-    // console.log("test case runs")
-    expect(res).to.have.status(201);
-    done();
-  });
+  it('should return status code 201 and a success message on successful creation', (done) => {
+//
+   
+
+chai.request(app).post('/user').send( {"fullName":"Shashwat Mohite",
+    "email": "shashwat@gmail.com",
```
Comment (DjangoUnchainedd) [inline test/test.js:85]: Use .env here too

File: models/allModels.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    createdAt: false,
+    updatedAt: false,
+    freezeTableName: true,
+    timestamps: false,
+    logging: false,
+  
+  });
+export default User;
+
+  User.hasOne(SellerProfile,{onDelete :'CASCADE', foreignKey : 'sellerId'});
```
Comment (DjangoUnchainedd) [inline models/allModels.js:200]: Test these relationships thoroughly.

File: connection.js (hunk 1 above)
Comment (smohite03) [reply connection.js:4]: ok

Comment (sgupta1409) [reply connection.js:4]: ok.

File: models/allModels.js (hunk 3 above)
Comment (sgupta1409) [reply models/allModels.js:200]: tested successfully.

**Pull request merged**